### ISSUE TITLE
keep strand

### ISSUE THREAD

**Issue by kullrich** [sourmash-bio/sourmash#2323]
Hi,
according to the manual at the moment, the reverse complement and in case of is_protein=True all 6 possible open reading frames are considered.
Is there an option to keep the strand and not do reverse complement or just keep 3/6 of possible ORFs?
Thank you in anticipation
Best regards
Kristian

**Comment by ctb** [comment]
hi @kullrich this is not possible at the moment but would not be hard for us to implement.

we could easily give you a script to produced sketches like this, and the rest of sourmash would work fine with the resulting sketch.  interested? (It just will take time to add that feature into `sourmash sketch`.)

see also https://github.com/sourmash-bio/sourmash/issues/657

**Comment by kullrich** [comment]
Yes, I am interested. Actually, I am re-implementing this (https://github.com/mrvollger/StainedGlass) but using sourmash jaccard distances instead of blast . By this, the tool is much faster and can be used without a snakemake environment. Anyhow, I need to keep track of the strand to be able to color possible rearrangements accordingly.
Best regards
Kristian

**Comment by ctb** [comment]
neat! will see what I can do :)

**Comment by kullrich** [comment]
Here, e.g. chr4 as a comparison, but with jaccard distances. I just need to work on the color map and to implement strand specific scores. Even multithreading might be possible to calcualte pairwise jaccard distances in parallel (maybe with dask?)
<img width="580" alt="CEN4_chr4_3926001_7255300_1000bp" src="https://user-images.githubusercontent.com/8100643/194941831-29137d6c-9d65-4a37-b190-cf7e72d868d0.png">
